package/src/lib.rs

```diff
@@ -22,9 +22,7 @@ impl Package {
     ///
     /// If multiple resources have the same path, the first one will be returned. You should try
     ///  to avoid this situation.
-    pub fn find<P>(&self, resource: P) -> Option<&'static [u8]> where P: AsPath {
-        let resource = resource.as_path();
-
+    pub fn find(&self, resource: &Path) -> Option<&'static [u8]> {
```

Comment:
The `AsPath` trait has been replaced by `AsRef<Path>`.
Could you put `pub fn find<P>(&self, resource: P) -> Option<&'static [u8]> where P: AsRef<Path> {` instead? This allows writing `find("hello")` for example.